lib/matplotlib/text.py

```diff
@@ -38,7 +38,7 @@
 
 
 def _process_text_args(override, fontdict=None, **kwargs):
-    "Return an override dict.  See :func:`~pyplot.text' docstring for info"
+    """Return an override dict.  See `~pyplot.text' docstring for info."""
```

Comment:
Style nit-picks: Since the primary function of triple quotes is to generate multi-line text blocks, I would prefer to see them used in docstrings only in the form
```
"""
This is a docstring that might have more than one line.
"""
```
If you want a docstring to occupy only a single line, to save space, then I think you should use quotes as in the line above before your change.  When I see triple quotes, I want to see the ending triple on a subsequent line; and starting a docstring with triple quotes followed by text on the same line looks ugly to me.

Reply:
This is actually explicitly specified in https://www.python.org/dev/peps/pep-0257/#one-line-docstrings.  If we decide to explicitly not follow that recommendation it should probably go into the documentation guidelines...

Reply:
Agreed.  I think that PEP is giving bad advice with a silly rationale, but it's not critical.  @tacaswell, do you want to follow the PEP for new code and whenever existing docstrings are edited?

Reply:
Maybe it's just habit, but using always triple quotes feels more consistent. Docstrings look more alike. Even though the rationale in PEP-257 is of course nonsense, I would stick to the suggested conventions.